modules/ding_content_rss/ding_content_rss.info

```diff
@@ -0,0 +1,25 @@
+name = Ding Content RSS
+description = Ding Content RSS feeds
+core = 7.x
+package = Ding!
+version = 7.x-4.1.0
```

Comment:
Normally we don't have the version tag in DDB CMS custom modules.

Reply:
This will be overriden on the next release, with the script you are using to create versions.